src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -426,7 +426,7 @@ namespace winrt::Microsoft::Terminal::Control::implementation
 
         if (imageSource == nullptr ||
             imageSource.UriSource() == nullptr ||
-            imageSource.UriSource().RawUri() != imageUri.RawUri())
+            !imageSource.UriSource().Equals(imageUri))

```

Comment:
`RawUri()` returns the string the URI was constructed with, not the normalized variant.
This can result in `imageSource.UriSource().RawUri()` being `file:///C:/...`, while `imageUri.RawUri()` is `C:\\...`.